### DISCUSSION TITLE
Unable to write new users into table.

### DISCUSSION THREAD

**Discussion by ChrisB007** [supabase/supabase#3346] [Questions]
Hello folks,
I have been trying to make supabase auth work for my app, but continue to run into a brick wall writing data into tables ?(user table). 
Here are the steps to reproduce the issue.
1. User click on the 'Magic Link', and upon clicking on the link and verifying via email  the user is written in the "Auth/User" table / page as expected. However, if you go to "Tables" page, nothing is stored in there. I am under the impression that supabase auto populate this page/ table when a new user is authenticated - Was I wrong?

**Comment by GaryAustin1** [comment]
The Tables UI is showing only tables in the public schema.  It does not show auth,storage,etc.  You can't access auth,storage directly from the client so if you need a users table with more info you will have to set it up using a trigger/function examples of which are in the docs and in issues here.

**Reply by ChrisB007** [reply]
Oh, I see. I was following this tutorial by the supabase team (https://www.youtube.com/watch?v=p561ogKZ63o&t=490s), fast forward to 45:04 mark -  The presenter's table was auto-populated with authenticated user's info. I was hoping that my app also behaves like that

**Reply by GaryAustin1** [reply]
Yes.  You have to create that profiles table then write a trigger and function to auto populate it when users is created.
